docker-compose.yml

```diff
@@ -1,75 +1,61 @@
-networks:
-    go-quai_default:
-        external: true
 services:
   prime:
+    env_file:
+      - network.env
```

Comment:
could you explain what this is for and all of the other occurrences? 

If that is the case then you will need to use the NETWORK specific ENV names like
PRIME_PORT_TCP ... etc.